### PR TITLE
Implement Faggot of the Day game

### PR DESCRIPTION
**Pull request by Kona-chan**
Advanced faggotry.

### CODE REVIEW

File: src/main/resources/hibernate.cfg.xml [hunk 1]
```diff
@@ -16,6 +16,7 @@
   <mapping class="me.rkfg.xmpp.bot.domain.Markov"/>
   <mapping class="me.rkfg.xmpp.bot.domain.MarkovFirstWord"/>
   <mapping class="me.rkfg.xmpp.bot.domain.MarkovFirstWordCount"/>
-  <mapping class="me.rkfg.xmpp.bot.domain.Opinion"/>
+  <mapping class="me.rkfg.xmpp.bot.domain.Opinion"/><mapping class="me.rkfg.xmpp.bot.domain.Contender" /><mapping
+ 		class="me.rkfg.xmpp.bot.domain.Winning" />
```
Comment (rkfg) [inline src/main/resources/hibernate.cfg.xml]: NEW LINE MOTHERFUCKER DO YOU SPEAK IT

Comment (Kona-chan) [inline src/main/resources/hibernate.cfg.xml]: ОНО САМО

File: src/main/java/me/rkfg/xmpp/bot/plugins/FaggotOfTheDayPlugin.java [hunk 2]
```diff
@@ -0,0 +1,294 @@
...
+import java.time.LocalTime;
+import java.time.ZoneId;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
+import java.util.Optional;
+import java.util.Random;
+import java.util.regex.Matcher;
+
+import javax.persistence.NonUniqueResultException;
```
Comment (rkfg) [inline src/main/java/me/rkfg/xmpp/bot/plugins/FaggotOfTheDayPlugin.java]: Use the Hibernate's one instead of this.

Comment (Kona-chan) [inline src/main/java/me/rkfg/xmpp/bot/plugins/FaggotOfTheDayPlugin.java]: Fug

**Pull request merged**